frontend/src/pages/Inbox/components/QuickReply.js

```diff
@@ -39,6 +37,7 @@ const QuickReply = ({
           message: reply.message,
           timestamp: new Date(),
         });
+        onClick();
```

Comment:
why don't you just add the webpage stuff here instead of passing a function?
```
webpage && selectWebpageType()
```

Reply:
That is Ruis code, just copy and paste in order to call the webpage.  Don't saw this, let me fix.